packages/apps/job-launcher/server/src/modules/webhook/webhook.dto.ts

```diff
@@ -31,12 +60,18 @@ export class WebhookDataDto {
   public chainId: ChainId;
 
   @ApiProperty({ name: 'escrow_address' })
+  @IsString()
+  @IsValidEthereumAddress()
```

Comment:
use the validator from `class-validator`

Reply:
changed